src/services/organization-extension.js

```diff
@@ -0,0 +1,108 @@
+// Dependencies
+const httpStatusCode = require('@generics/http-status')
+const common = require('@constants/common')
+const orgExtensionQueries = require('@database/queries/organizationExtensions')
+const reviewStageQueries = require('@database/queries/reviewStage')
+const entityTypeQueries = require('@database/queries/entityType')
+const { UniqueConstraintError } = require('sequelize')
+const responses = require('@helpers/responses')
+const _ = require('lodash')
+module.exports = class orgExtensionsHelper {
+	/**
+	 * Create Organization Config.
+	 * @method
+	 * @name createConfig
+	 * @param {Object} bodyData - Organization Config body data.
+	 * @returns {JSON} - Organization Config created response.
+	 */
+
+	static async createConfig(bodyData, organization_id) {
+		try {
+			bodyData.organization_id = organization_id
+			const { resource_type } = bodyData
+
+			const resources = await entityTypeQueries.findOneEntityTypeAndEntities({
+				organization_id: organization_id,
+				value: common.RESOURCES,
+			})
+
+			const resourceList = resources.entities
+			const validResourceTypes = _.map(resourceList, 'value')
+			if (!validResourceTypes.includes(resource_type)) {
+				return responses.failureResponse({
+					message: `resource_type ${resource_type} is not a valid`,
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+
+			if (bodyData.review_type === common.REVIEW_TYPE_SEQUENTIAL && bodyData.review_stages) {
+				//review stage creation
+				const reviewStages = bodyData.review_stages.map((stage) => ({
+					...stage,
+					organization_id,
+					resource_type,
+				}))
+
+				await reviewStageQueries.bulkCreate(reviewStages)
```

Comment:
How are we getting the roles for review stages? @priyanka-TL

Reply:
from the req body.

Reply:
I think it is not added in the api doc. can we add that? @priyanka-TL

Reply:
@adithyadinesh0412 its added already